modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsEpics.scala

```diff
@@ -28,361 +28,750 @@ import scala.concurrent.duration.FiniteDuration
  * Created by jluhrs on 10/1/15.
  */
 
-final class TcsEpics[F[_]: Async](epicsService: CaService, tops: Map[String, String]) {
+trait TcsEpics[F[_]] {
```

Comment:
Converted `TcsEpics` into a `trait`. This allows to write a special version for unit tests.